docs/content/misc/math-expr.md

```diff
@@ -0,0 +1,34 @@
+---
+layout: doc_page
+---
+
+This expression language supports following operators (listed in increasing order of precedence).
+
+|Operators|Description|
+|---------|-----------|
+|and,or|Binary Logical AND, OR|
+|<, <=, >, >=, ==, !=|Binary Comparison|
+|+, -|Binary additive|
+|*, /, %|Binary multiplicative|
+|^|Binary power op|
+|!, -|Unary NOT and Minus|
+
+long and double data types are supported, number containing a dot is interpreted as a double or else a long. That means, always add a '.' to your number if you want it intepreted as a double value.
```

Comment:
less confusing wording: 
Long and double data types are supported.  If a number contains a dot, it is interpreted
as a double, otherwise it is interpreted as a long.